docs/user_guide/further_resources.rst

```diff
@@ -18,6 +18,7 @@ Books
 Papers
 ------
 
+- `Fairlearn: A toolkit for assessing and improving fairness in AI <https://www.microsoft.com/en-us/research/uploads/prod/2020/05/Fairlearn_whitepaper.pdf>`_ (Sarah Bird, Miroslav Dudík, Hanna Wallach and Kathleen Walker, 2020)
```

Comment:
nit: could you keep the style the same as in the others? That would be "(Bird et al., 2020)